test/integration_tests/test_load_executor.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import os
+import cv2
+import numpy as np
+import pandas as pd
+
+from src.parser.parser import Parser
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+from src.optimizer.plan_generator import PlanGenerator
+from src.executor.plan_executor import PlanExecutor
+from src.catalog.catalog_manager import CatalogManager
+from src.storage import StorageEngine
+from src.models.storage.batch import Batch
+
+
+NUM_FRAMES = 10
+
+
+class LoadExecutorTest(unittest.TestCase):
+
+    def create_sample_video(self):
```

Comment:
it looks like this function can be reused across multiple test cases. Should we move it a different file (like `src/utils/`)?

Reply:
Sure we can move it to the test utils